src/framework/theme/components/checkbox/checkbox.component.ts

```diff
@@ -130,13 +130,14 @@ import { convertToBoolProperty } from '../helpers';
     <label class="label">
       <input type="checkbox" class="native-input visually-hidden"
              [disabled]="disabled"
-             [checked]="value"
+             [checked]="checked"
              (change)="updateValueAndIndeterminate($event)"
              (blur)="setTouched()"
+             (click)="$event.stopPropagation()"
              [indeterminate]="indeterminate">
-      <span [class.indeterminate]="indeterminate" [class.checked]="value" class="custom-checkbox">
+      <span [class.indeterminate]="indeterminate" [class.checked]="checked" class="custom-checkbox">
         <nb-icon *ngIf="indeterminate" icon="minus-bold-outline" pack="nebular-essentials"></nb-icon>
-        <nb-icon *ngIf="value && !indeterminate" icon="checkmark-bold-outline" pack="nebular-essentials"></nb-icon>
+        <nb-icon *ngIf="checked && !indeterminate" icon="checkmark-bold-outline" pack="nebular-essentials"></nb-icon>
```

Comment:
how did we achieve the expected result by simply renaming the variable?

Reply:
@nnixaa Sorry, it's not related to the fix. I decided to remove usage of the deprecated `value` property in the scope of this PR as well. The main change is on line 136.

Reply:
right, missed it